SubZero/src/model/filter/RGBFilter.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * RGBFilter.cpp
+ *
+ *  Created on: Feb 24, 2015
+ *      Author: ahsueh1996
+ */
+
+#include "RGBFilter.h"
+
+RGBFilter::RGBFilter(int r, int g, int b) {
+	setValues(r,g,b);
+}
+
+RGBFilter::~RGBFilter() {
+	// TODO Auto-generated destructor stub
+}
+
+int RGBFilter::filter (CamData * inputImg) {
+	return 0;
+}
+
+
+void RGBFilter::setValues(int r, int g, int b){
+	this -> r = r;
+	this -> g = g;
+	this -> b = b;
+}
```

Comment:
Good point.  For that specific example, I usually like to omit those spaces since they don't add too much in terms of readability and just takes up space.  Maybe next meeting we can identify some of these and maybe revisit our old meeting minutes that briefly discussed some of these